decisions/versioning-rules.md

```diff
@@ -0,0 +1,32 @@
+<!-- Copy this to make your own decisions. -->
+
+# Versioning approach for component releases
+
+## TL;DR
+<!-- Only if long enough to merit one. -->
+
+## Decision
+When a new Backpack component is to be released the approach is to use semver to manage versions which are based on the types of changes that have been made to each component. To ensure that changes made don't cause unexpected outages or issues throughout the applications the following reasoning should be followed.
```

Comment:
Can you make the word `semver` be a link to https://semver.org/? E.g. [semver](https://semver.org/).